docs/plot/environment.yml

```diff
@@ -5,5 +5,4 @@ dependencies:
   - descartes
   - geojson
   - matplotlib
-  - proj
```

Comment:
that change alone will not bring https://github.com/OSGeo/PROJ/pull/2981
you should rather have done:
```
git fetch origin master
git rebase origin/master
```

Reply:
Thanks. Having git issues, but hopefully solved now.